packages/contracts/contracts/core/component/DAOPermissioned.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
+
+import "../permission/PermissionManager.sol";
+import "./../IDAO.sol";
+
+/// @title DAOPermissioned
+/// @author Aragon Association - 2022
+/// @notice An abstract contract providing a meta transaction compatible modifier to make functions permissioned through an associated DAO.
+/// This contract provides an `auth` modifier that can be applied to functions in inheriting contracts. The permission to call these functions is managed by the associated DAO.
+/// @dev Make sure to call `__DAOPermissioned_init` during initialization of the inheriting contract.
+///      This contract is compatible with meta transactions through OZ's `ContextUpgradable`.
+abstract contract DAOPermissioned is Initializable, ContextUpgradeable {
```

Comment:
This is to make clear that the inheriting contract is permissioned through a managing DAO. 
With the old name, I think you could be misled to think that the inheriting contract has its own way how to manage permissions. 
I wanted to prevent this by clearer naming, but I am also not so happy with this outcome.

Reply:
@novaknole Jordi and I agreed on `DaoAuthorizable` because this provides the `IDAO dao` member and the `auth` modifier

Reply:
For the discussion if this should be abstract or not, let's make a separate task.